integration/spark/shared/src/main/java/io/openlineage/spark/agent/facets/EnvironmentFacet.java

```diff
@@ -15,7 +15,7 @@
  * cluster used, reporting certain environment variables, or resolving mount points.
  */
 public class EnvironmentFacet extends OpenLineage.DefaultRunFacet {
-  @JsonProperty("environment-properties")
+  @JsonIgnore
```

Comment:
why is this necessary?

Reply:
For now, the environment-properties facet looks something like this 
```
"environment-properties": {
  "_producer": "https://github.com/OpenLineage/OpenLineage/tree/1.13.0-SNAPSHOT/integration/spark",
  "_schemaURL": "https://openlineage.io/spec/2-0-2/OpenLineage.json#/$defs/RunFacet",
  "environment-properties": {
    "foo": "...",
    "bar": "..."
  }
}
```
This change will just make a cleaner JSON:
```
"environment-properties": {
  "_producer": "https://github.com/OpenLineage/OpenLineage/tree/1.13.0-SNAPSHOT/integration/spark",
  "_schemaURL": "https://openlineage.io/spec/2-0-2/OpenLineage.json#/$defs/RunFacet",
  "foo": "...",
  "bar": "..."
}
```
Please let me know if it would be okay to keep the change. If not, I will revert it.

Reply:
I have removed this and moved to a different approach for reporting facets.